README.md

```diff
@@ -1,12 +1,53 @@
-## How to set navigation
-- Create a directory hierarchy in `src/pages/config.md`
+# Adobe Connect SDK Documentation
 
-## Launching a deploy
-- Go to Actions > Deployment > Run workflow
+Adobe Connect SDK documentation site deployed to EDS.
+The production address is https://developer.adobe.com/adobe-connect-sdk/
 
-## URL
-- developer-stage.adobe.com/{path-prefix}/{relative-path-to-file}
+## Quick Start
 
-## Where to ask for help
+Start development server:
+```bash
+npm run dev
+```
+Runs Express server on port 3003 serving files from `src/pages/`.
 
-The slack channel #adobe-developer-website is our main point of contact for help. Feel free to join the channel and ask any questions.
+## Commands
+
+**Development**
+- `npm run dev` - Start local server
+
+**Content Management**
+- `npm run buildNavigation` - Generate navigation structure
+- `npm run buildRedirections` - Build URL redirections  
+- `npm run renameFiles` - Rename files to Adobe conventions
+- `npm run normalizeLinks` - Normalize internal/external links
+
+**Validation**
+- `npm run lint` - Run linting checks
+
+**Site Features**
+- `npm run buildSiteWideBanner` - Generate site-wide banner
+
+*All commands use `@AdobeDocs/adp-devsite-utils` for standardized tooling.*
+
+## Linting
+
+**Automated**: Runs on PRs when `src/pages/**` files change
+**Manual**: `npm run lint`
+
+Validates markdown syntax, links, content structure, and Adobe style guidelines.
+
+## Navigation
+
+1. Edit directory hierarchy in `src/pages/`
+2. Update `src/pages/config.md` 
+3. Run `npm run buildNavigation`
+
+## Deployment
+
+**Staging**: Actions > Deployment > Run workflow  
+**URL**: `developer-stage.adobe.com/adobe-connect-sdk/`
+
```

Comment:
There should be a note about incremental builds from the last commit or use the `deployAll` function. Also, deploying from a branch to `stage`.